AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -600,13 +609,125 @@ def hair(property: str = 'name') -> str:
           - 'requires' a list of component names as strings required by this component.
             Use editor_entity_utils EditorEntity.add_components(list) to add this list of requirements.
           - 'Hair Asset' Asset.id of the hair TressFX asset.
+          - 'Enable Area Lights' (bool default True)
+          - 'Enable Azimuth' Azimuth Contribution (bool default True)
+          - 'Enable Directional Lights' (bool default True)
+          - 'Enable IBL' imaged-based lighting for hair (bool default True)
+          - 'Enable Longitude' Longitude Contribution (bool default True)
+          - 'Enable Marschner R' (bool default True)
+          - 'Enable Marschner TRT' (bool default True)
+          - 'Enable Marschner TT' (bool default True)
+          - 'Enable Punctual Lights' (bool default True)
+          - 'Enable Shadows' (bool default True)
+          - 'Hair Lighting Model' from atom_constants.py HAIR_LIGHTING_MODEL (default 'Marschner')
```

Comment:
What is the file type this field accepts? 

Was expecting: (File type XYZ, default 'Marschner'"

Reply:
its an enum for the algorithm rather than a model asset. So it's not a file but selecting from a list of simulation systems.

Reply:
updating the description to "simulation algorithm from atom_constants.py"

Reply:
ah cool, makes sense, thanks for clarifying!